publication-event-handlers/src/main/java/no/unit/nva/publication/events/bodies/DoiMetadataUpdateEvent.java

```diff
@@ -128,8 +127,8 @@ private static URI extractCustomerId(Publication publication) {
         return Optional.ofNullable(publication.getPublisher()).map(Organization::getId).orElse(null);
     }
 
-    private static URI inferPublicationId(Publication newEntry) {
-        return UriWrapper.fromUri(NVA_API_DOMAIN)
+    private static URI inferPublicationId(Publication newEntry, String apiHost) {
+        return UriWrapper.fromHost(apiHost)
                    .addChild("publication")
                    .addChild(newEntry.getIdentifier().toString())
                    .getUri();
```

Comment:
NVA_API_DOMAIN has never been provided in environment to UpdatePublicationHandler. 
```
See log:
java.lang.ExceptionInInitializerError: java.lang.ExceptionInInitializerError
java.lang.ExceptionInInitializerError
	at no.unit.nva.publication.events.bodies.DoiMetadataUpdateEvent.inferPublicationId(DoiMetadataUpdateEvent.java:132)
	at no.unit.nva.publication.events.bodies.DoiMetadataUpdateEvent.createUpdateDoiEvent(DoiMetadataUpdateEvent.java:56)
```